src/encord_active/lib/dataset/outliers.py

```diff
@@ -6,8 +7,15 @@
 from encord_active.lib.metrics.utils import MetricSchema
 
 
+class OutlierStatus(str, Enum):
```

Comment:
[nitpick] 💅

This can just be `Severity` and we could reuse it to categorize other insights in the future (not just outliers)

```suggestion
class Severity(str, Enum):
```